release-notes.txt

```diff
@@ -0,0 +1,172 @@
+onl: fixup comments about installed libraries
+linux-yocto-onl: import linux-yocto dependencies
+linux-yocto-onl/x86-intel: disable I2C HID support
+linux-yocto-onl/armel-iproc: reenable strong stackprotector
+linux-yocto-onl: enable KCONF_BSP_AUDIT
+linux-yocto-onl: drop old ONL config option
+linux-yocto-onl: update kernel config for 5.15
+linux-yocto-onl: update kernel config for 5.14
+linux-yocto-onl: update kernel config for 5.13
+linux-yocto-onl: update kernel config for 5.12
+linux-yocto-onl: update kernel config for 5.11
+linux-yocto-onl: update kernel config for 5.10
+linux-yocto-onl: update kernel config for 5.9:
+linux-yocto-onl: update kernel config for 5.8
+linux-yocto-onl: update kernel config for 5.7
+linux-yocto-onl: update kernel config for 5.6
+linux-yocto-onl: update kernel config for 5.5
+linux-yocto-onl: update kernel config for 5.4
+linux-yocto-onl: update kernel config for 5.3
+linux-yocto-onl: update kernel config for 5.2
+linux-yocto-onl: update kernel config for 5.1
+linux-yocto-onl: update kernel config for 5.0
+linux-yocto-onl: update kernel config for 4.20
+linux-yocto-onl: update kernel config for 4.19
+linux-yocto-onl: update kernel config for 4.18
+linux-yocto-onl: update kernel config for 4.17
+linux-yocto-onl: update kernel config for 4.16
+linux/yocto-onl: update kernel config for 4.15
+onl: delta-ag5648: fix build with GCC 10
+onl: cel-questone-2a: fix compilation with GCC 10+
+onl: update python3 conversion to work with python 3.10
+onl: fix build with PyYAML 6.0
+onl: fix compilation with GCC 10
+make-mod-scrips: drop obsolete append
+linux-yocto-onl: drop local definitions of functions
+linux-yocto-onl: drop our virtualization include
+strace: drop local version
+conf: update tune includes to new location
+update layer to kirkstone
+onl: ship onlpd instead of onlpdump
+kernel: update source comments in linux-yocto-onl_5.15.bb
+conf: drop BISDN_ONIE_PLATFORM now that it is autogenerated
+onl: move ONL_DEBIAN_SUITE into recipe
+conf: drop now unused ONL_ARCH
+onl: calculate ONL_BUILD_ARCH dynamically
+conf/machine: drop ONL_PLATFORM_SUBDIR
+onl.inc: drop unused PLATFORM_PATH
+onl: use the onl package manager for resolving paths
+onl: update python3 conversion patch
+conf: fixup override usage
+linux-yocto-onl: do not set CONFIG_IP_NF_FILTER
+linux-yocto-onl: do not set CONFIG_NET_UDP_TUNNEL
+linux-yocto-onl: x86-intel: do not set CONFIG_CGROUP_HUGETLB
+linux-yocto-onl: x86-intel: do not set CONFIG_VXLAN
+linux-yocto-onl: use kernel-meta for ext4
+linux-yocto-onl: armel-iproc: use ext4 module for ext2 and ext3
+linux-yocto-onl: armel-iproc: enable FS_SECURITY for ext4
+linux-yocto-onl: armel-iproc: enable ACL support ext4
+onl: make use of yocto's python3-native
+yocto-linux-onl: armel-iproc: do not disable VXLAN
+
+meta-switch:
+Revert "frr: update to 8.3"
+systemd: let any interface satisfy systemd-networkd-wait-online
+bump distro to version 3.7.0
+baseboxd: bump to 1.12.0
+frr: update to 8.3
```

Comment:
I would prefer to just delete both commit messages regarding 8.3 (commit and revert), since otherwise people might only see the first one and be a bit confused.

Reply:
I'd rather not "hand-tune" autogenerated things. You will also see it when you look at the git history yourself, so I think having it is fine.

Reply:
I'd rather not cofuse people that read this and do not know that this is auto generated.

Reply:
Or is this not not exatly the same that we will upload to the repos?

Reply:
How about we change the title to "Detailed changelog (auto generated from git history)" or so? This really should be a fire and forget thing, and not be touched up by anything. Any manual changes will also get likely be dropped/missed in case we need to regenerate it.

Reply:
We changed this a couple of times manually before, to avoid confusing people or to redact certain internal information. Lets just remove the FRR 8.3 here, so it does not cause any confusion for people grepping through the changelog.

Reply:
I still disagree, but changed it to not getting hung up on this.